src/com/company/MyLinkedList.java

```diff
@@ -99,14 +99,12 @@ public void insertAfter(LinkedListNode nodeAfterWhichIsInserted, LinkedListNode
             temp = temp.getNextNode();
         }
 
-        if(temp.getNextNode()!=null){
-               nodeBeforeWhichIsInserted = temp.getNextNode();
-            }
+        if (temp.getNextNode() != null) {
+            nodeBeforeWhichIsInserted = temp.getNextNode();
+        }
```

Comment:
you can remove the line 96, and replace line 102 - 104 with:

LinkedListNode nodeBeforeWhichIsInserted = temp.getNextNode();

Nothing bad is going to happen from rewriting null with null and it makes your code more compact